apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx

```diff
@@ -0,0 +1,325 @@
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { Field, Wizard, WizardPage } from "@wso2is/form";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import {
+    HyprAuthenticationProviderCreateWizardFormValuesInterface
+} from "./hypr-authentication-provider-create-wizard";
+import { getIdentityProviderList } from "../../../api";
+import { IdentityProviderManagementConstants } from "../../../constants";
+import { IdentityProviderListResponseInterface, IdentityProviderTemplateInterface } from "../../../models";
+import { handleGetIDPListCallError } from "../../utils";
+
+/**
+ * Proptypes for the HyprAuthenticationWizardFrom.
+ */
+interface HyprAuthenticationProviderCreateWizardContentPropsInterface extends IdentifiableComponentInterface {
+    /**
+     * Trigger form submit.
+     * @param submitFunctionCallback - Callback.
+     */
+    triggerSubmission: (submitFunctionCallback: () => void) => void;
+    /**
+     * Trigger previous page.
+     * @param previousFunctionCb - Callback.
+     */
+    triggerPrevious: (previousFunctionCb: () => void) => void;
+    /**
+     * Callback to change the wizard page,
+     * @param pageNo - Page Number.
+     */
+    changePageNumber: (pageNo: number) => void;
+    /**
+     * IDP template.
+     */
+    template: IdentityProviderTemplateInterface;
+    /**
+     * Total wizard page count.
+     * @param pageCount - Page number.
+     */
+    setTotalPage: (pageCount: number) => void;
+    /**
+     * Callback to be triggered for form submit.
+     * @param values - Form values.
+     */
+    onSubmit: (values: HyprAuthenticationProviderCreateWizardFormValuesInterface) => void;
+}
+
+const FORM_ID: string = "hypr-authenticator-wizard-form";
+
+/**
+ * Hypr Authentication Provider Create Wizard content component.
+ *
+ * @param props - Props injected to the component.
+ * @returns Functional component.
+ */
+export const HyprAuthenticationProviderCreateWizardContent: FunctionComponent<
+    HyprAuthenticationProviderCreateWizardContentPropsInterface
+> = (
+    props: HyprAuthenticationProviderCreateWizardContentPropsInterface
+): ReactElement => {
+
+    const {
+        triggerSubmission,
+        triggerPrevious,
+        changePageNumber,
+        template,
+        setTotalPage,
+        onSubmit,
+        [ "data-componentid" ]: componentId
+    } = props;
+
+    const { t } = useTranslation();
+
+    const [ idpList, setIdPList ] = useState<IdentityProviderListResponseInterface>({});
+    const [ isIdPListRequestLoading, setIdPListRequestLoading ] = useState<boolean>(undefined);
+
+    /**
+     * Loads the identity provider authenticators on initial component load.
+     */
+    useEffect(() => {
+
```

Comment:
```suggestion
```

Reply:
shouldn't we leave a blank line after function signature?